Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -462,36 +468,64 @@ def get_family_info_command(intezer_api: IntezerApi, args: dict) -> CommandResul
 
 # region Enrich DBot
 
-def enrich_dbot_and_display_file_analysis_results(intezer_result):
+def enrich_dbot_and_display_file_analysis_results(intezer_result: dict, file_metadata: dict):
```

Comment:
Please add the return type:

```suggestion
def enrich_dbot_and_display_file_analysis_results(intezer_result: dict, file_metadata: dict) -> CommandResults:
```